.github/workflows/program-auction-house.yml

```diff
@@ -123,4 +123,4 @@ jobs:
         run: |
           cargo +${{ env.RUST_TOOLCHAIN }} test -- --nocapture --test-threads 1
           cargo +${{ env.RUST_TOOLCHAIN }} test-bpf --version
-          cargo +${{ env.RUST_TOOLCHAIN }} test-bpf --bpf-out-dir ../../target/deploy/ --test ${{ env.TEST }} -- --nocapture --test-threads 1
+          cargo +${{ env.RUST_TOOLCHAIN }} test-bpf --bpf-out-dir ../../test-programs/ --test ${{ env.TEST }} -- --nocapture
```

Comment:
I'd prefer to run tests in series in CI as otherwise it gets harder to identify an issue for 2 reasons:

- test output of different tests is intermingled
- tests might affect each other in odd ways

Also given that running the tests takes a very small percentage of the time compared to the setup needed before that I believe the gain doesn't outweigh the potential issues here.

Reply:
Tests are supposed to be independent. What are the potential ways tests might affect each other?

Reply:
I am not sure, I suppose that would only be a problem if they were running against the same validator or similar.

However the first point is actually at least equally important .. running tests in parallel leads to output that makes it very hard to piece together information of a failing test

Reply:
Actually running tests in parallel also could lead to timing issues due to the host box getting overloaded and thus tests timing out esp. considering recent block hashes and such.

Reply:
Ok, I'll fix this, thanks.